deploy_hub.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+TAG="${TRAVIS_TAG:-latest}"
+
+_logout() {
+  docker logout
+}
+
+trap _logout EXIT TERM
+
+docker login -u "${DOCKER_HUB_USERNAME}" -p "${DOCKER_HUB_PASSWORD}"
+docker build -t "${DOCKER_HUB_USERNAME}/mastodon:${TAG}" .
+docker tag "${DOCKER_HUB_USERNAME}/mastodon:${TAG}" "${DOCKER_HUB_USERNAME}/mastodon:latest"
+docker push "${DOCKER_HUB_USERNAME}/mastodon:${TAG}"
+docker push "${DOCKER_HUB_USERNAME}/mastodon:latest"
```

Comment:
I do not think the last line is necessary.

Reply:
It is if `TRAVIS_TAG` is actually set. You would want a tagged release to become `latest` as well. Also, this is a no-op in the daemon if it detects that the digest has already been pushed.